config/cloudinary.config.js

```diff
@@ -0,0 +1,10 @@
+const cloudinary = require('cloudinary').v2;
```

Comment:
you can just leave a file name as cloudinary, "config" is redundant since it is already in the config folder.

Reply:
fixed ✅